README.md

```diff
@@ -227,6 +227,19 @@ The below example is for v1
 client.update_service service1
 ```
 
+#### Patch an entity (by name)
+For example: `patch_service "service name"`, `patch_pod "pod name"`, `patch_replication_controller "rc name"`, `patch_secret "secret name"`, `patch_resource_quota "resource quota name"`, `patch_limit_range "limit range name"`, `patch_persistent_volume "persistent volume name"`, `patch_persistent_volume_claim "persistent volume claim name"`, `patch_component_status "component name"`, `patch_service_account "service account name"`
```

Comment:
@abonas, changed it to method name only, while the required and optional parameters are specified below (same as the description for update entity). If you still find it unnecessary I can remove it too.  cc @simon3z